app/javascript/views/groupUsersList/GroupUserList.test.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import GroupUsersList from './GroupUsersList'
+
+describe('GroupUserList', () => {
+  let wrapper
+  let mockFetchGroupUsersList
+  const mockFilter = [{ key1: 'some key' }]
+  const location = { filter: { query: mockFilter } }
+
+  beforeEach(() => {
+    mockFetchGroupUsersList = jest.fn()
+
+    wrapper = shallow(
+      <GroupUsersList
+        location={location}
+        actions={{
+          fetchGroupUsersList: mockFetchGroupUsersList,
+        }}
+      />,
+      {
+        disableLifecycleMethods: true,
+      }
+    )
+  })
+
+  describe('#componentDidMount', () => {
+    it('componentDidMount is called', () => {
+      shallow(
+        <GroupUsersList
+          location={location}
+          actions={{
+            fetchGroupUsersList: mockFetchGroupUsersList,
+          }}
+        />
+      )
+      expect(mockFetchGroupUsersList).toHaveBeenCalledWith(mockFilter)
+    })
+
+    it('componentDidMount is called, FetchGroupUsersList is not called', () => {
```

Comment:
```suggestion
    it('componentDidMount is called with empty location, FetchGroupUsersList is not called', () => {
```

Is this the circumstance you are testing?

Reply:
Done. Renamed in accordance to suggestion